File: src/main/java/dev/lonami/uniffidl/psi/UdlTokenSets.java
package dev.lonami.uniffidl.psi;

import com.intellij.psi.tree.TokenSet;

public interface UdlTokenSets {
    TokenSet IDENTIFIERS = TokenSet.create(UdlTypes.IDENTIFIER);
    TokenSet COMMENTS = TokenSet.create(UdlTypes.COMMENT);
    TokenSet STRINGS = TokenSet.create(UdlTypes.STRING);
}

File: src/main/kotlin/dev/lonami/uniffidl/psi/UdlTokenSets.kt
package dev.lonami.uniffidl.psi

import com.intellij.psi.tree.TokenSet

object UdlTokenSets {
    @JvmField
    val IDENTIFIERS: TokenSet = TokenSet.create(UdlTypes.IDENTIFIER)
    @JvmField
    val COMMENTS: TokenSet = TokenSet.create(UdlTypes.COMMENT)
    @JvmField
    val STRINGS: TokenSet = TokenSet.create(UdlTypes.STRING)
}
